src/wizard/wizard.scss

```diff
@@ -2,168 +2,147 @@
-          background-color: t(iui-color-background-1);
-          color: t(iui-text-color-muted);
+        .iui-circle {
+          align-items: center;
+          border-radius: 50%;
+          display: flex;
+          flex-shrink: 0;
+          width: $iui-l;
+          height: $iui-l;
+          justify-content: center;
+          overflow-wrap: break-word;
+          padding: 0;
+          position: relative;
+          text-align: center;
+          user-select: none;
+          transition: background-color $iui-speed-fast ease-out, border-color $iui-speed-fast ease-out, color $iui-speed-fast ease-out;
+          @include themed {
+            border: 1px solid t(iui-color-background-5);
+            background-color: t(iui-color-background-1);
+            color: t(iui-text-color-muted);
+          }
         }
       }
-    }
-
-    > .iui-wizards-step-track,
-    > .iui-wizards-step > .iui-wizards-step-track {
-      height: $iui-xxs;
-      position: absolute;
-      top: $iui-l * 0.5;
-      z-index: 0;
 
-      &.iui-wizards-step-track-before,
-      &.iui-wizards-step-track-after,
-      &.iui-wizards-step-track-main {
+      .iui-step-name {
+        text-align: center;
+        width: 100%;
+        user-select: all;
         @include themed {
-          background-color: t(iui-color-background-5);
+          color: t(iui-text-color-muted);
         }
       }
 
-      &.iui-wizards-step-track-before,
-      &.iui-wizards-step-track-after {
-        width: 50%;
-      }
-
-      &.iui-wizards-step-track-before {
-        left: 0;
-      }
-
-      &.iui-wizards-step-track-after {
-        right: 0;
-      }
-
-      &.iui-wizards-step-track-main {
-        display: block;
-        flex-basis: fill;
-        flex-grow: 1;
-        overflow: visible;
-        position: relative;
-      }
-
-      &.iui-wizards-step-track-filled {
-        @include themed {
-          background-color: t(iui-color-foreground-positive) !important;
+      &.iui-current {
+        .iui-step-name {
+          @include themed {
+            color: t(iui-color-foreground-positive);
+          }
+          font-weight: $iui-font-weight-semibold;
         }
-      }
-    }
-
-    > .iui-wizards-step-current {
-      > .iui-wizards-step-title {
-        @include themed {
-          color: t(iui-color-foreground-positive);
+  
+        .iui-circle {
+          font-weight: $iui-font-weight-semibold;
+          @include themed {
+            color: t(iui-color-foreground-accessory);
+            border-color: t(iui-color-background-positive);
+            background-color: t(iui-color-background-positive);
+          }
         }
-        font-weight: $iui-font-weight-semibold;
-      }
-
-      > .iui-wizards-step-circle {
-        @include themed {
-          color: t(iui-color-foreground-accessory);
-          border: 1px solid t(iui-color-background-positive);
-          background-color: t(iui-color-background-positive);
+  
+        .iui-before {
```

Comment:
I couldn't found it being used.

Reply:
`.iui-before` is removed in the next push.